packages/codegen-ui/lib/__tests__/generate-form-definition/helpers/form-field.test.ts

```diff
@@ -469,13 +469,12 @@ describe('getFormDefinitionInputElement', () => {
     const config = {
       inputType: {
         type: 'CheckboxField',
-        defaultChecked: true,
       },
     };
 
     expect(getFormDefinitionInputElement(config)).toStrictEqual({
       componentType: 'CheckboxField',
-      props: { label: 'Label', name: 'fieldName', value: 'fieldName', defaultChecked: true },
+      props: { label: 'Label', name: 'fieldName', value: 'fieldName' },
```

Comment:
There was a certain reason we were using `defaultChecked` but I don't remember why. Were you able to test both Create and Update forms to ensure there are no regressions?

Reply:
Yeah I was very concerned with regression so I've tested both Create and Update forms with checkboxes.  In both scenarios, the initial checkbox state is shown correctly, and create/update actions contain the correct checkbox state.